modules/gltf/gltf_document.cpp

```diff
@@ -464,7 +477,7 @@ String GLTFDocument::_gen_unique_name(Ref<GLTFState> state, const String &p_name
 		name = s_name;
 
 		if (index > 1) {
-			name += " " + itos(index);
+			name += itos(index);
```

Comment:
Removed the trailing space to align with the behavior used by the editor for Nodes (which I suspect would be aligned with user expectations)

Reply:
This case aligning with user expectation makes sense.

Reply:
👍